salt/modules/yumpkg.py

```diff
@@ -896,7 +896,13 @@ def refresh_db(**kwargs):
     branch_arg = _get_branch_option(**kwargs)
 
     clean_cmd = [_yum(), '--quiet', 'clean', 'expire-cache']
-    update_cmd = [_yum(), '--quiet', 'check-update']
+
+    if __grains__.get('os_family') == 'RedHat' and __grains__.get('osmajorrelease') == '7':
+        # This feature is disabled because it is not used by Salt. Furthermore, this feature lasts a lot with using large repo like EPEL
+        update_cmd = [_yum(), '--quiet', '--setopt=autocheck_running_kernel=false', 'check-update']
```

Comment:
why not just make this

`update_cmd.append('--setopt=autocheck_running_kernel=false')`

Instead of having to set update_cmd twice.